uint16 counter
uint16 time
uint16 A2D_1
uint16 A2D_2
uint16 A2D_3
uint16 A2D_4
uint16 A2D_5
uint16 A2D_6
uint16 A2D_7
uint16 A2D_8
uint16 Laser
uint8 PWM1
uint8 PWM2
uint8 PWM3
uint8 PWM4
uint16 Radio_range
uint16 Radio_range_conf
uint8 Configuration
uint16 Stepper
uint8 status